src/main/java/uk/gov/register/core/TransactionalMemoizationStore.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.register.core;
+
+import uk.gov.verifiablelog.store.memoization.MemoizationStore;
+
+public interface TransactionalMemoizationStore extends MemoizationStore {
```

Comment:
does this need to be an interface? is there any change volatility around this that we are going to have multiple implementations of it?